.github/workflows/dotnet-core.yml

```diff
@@ -1,21 +1,24 @@
 name: .NET Core
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push, pull_request]
 
 jobs:
-  build:
-    strategy:
-      matrix:
-        platform: [ windows-latest, ubuntu-latest ]
-    
-    runs-on: ${{ matrix.platform }}
+  ci:    
+    runs-on: windows-latest
 
     steps:
     - uses: actions/checkout@v2
+    - name: Fetch tags
+      run: |
+        git fetch origin +refs/tags/*:refs/tags/*
+        git fetch --prune --unshallow
+    - name: Build Reason
+      run: "echo ref: ${{github.ref}} event: ${{github.event_name}}"
+    - name: Build Version
```

Comment:
I think this part isn't necessary, because MinVer runs with `dotnet build`

Reply:
I'd like to clarify on the unnecessary part, are we talking about `Build Reason` phase? I am fine removing it, but keeping it helps understanding essence of what the trigger was and also co-relate what the workflow would do. I'd be fine removing it in case there's an alternate to the above mentioned information.

Reply:
Sorry about the confusion. I was referring to the minver action. The action is meant to be used for projects other than dotnet. I think it doesn't do anything.

I should remove it from LtiLibrary as well.

Reply:
Sounds good. Removing it right away.